src/main/java/gift/Product.java

```diff
@@ -0,0 +1,53 @@
+package gift;
```

Comment:
패키지를 나눠주세요. gift.model 이런식으로요  
```suggestion
package gift.model;
```